plinth/modules/help/help.py

```diff
@@ -71,3 +71,14 @@ def manual(request):
         request, 'help_manual.html',
         {'title': _('{box_name} Manual').format(box_name=_(cfg.box_name)),
          'content': content})
+
+def status_logs(request):
+    X = 100
```

Comment:
Variable names should be as descriptive as possible (without being too verbose), I'd propose something like 'nr_lines' instead.
Also, upper case variables are not PEP compliant - please check your code with a validation tool like pylint or flake8.